site/content/en/docs/Getting Started/om_basics.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Understand Open Match"
 linkTitle: "Basic Concepts"
```

Comment:
Rename this to 'core concepts'?

Reply:
https://github.com/googleforgames/open-match-docs/issues/22